<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UITest</class>
 <widget class="QWidget" name="UITest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="te_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>161</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_enter">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>720</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_backup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>561</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
